ovh/resource_domain_name_servers.go

```diff
@@ -0,0 +1,220 @@
+package ovh
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/ovh/go-ovh/ovh"
+	"log"
+	"strings"
+)
+
+type OvhDomainNameServerUpdate struct {
```

Comment:
could you place the structs in a file `type_domain.go`, to keep consistency with other resources ?